### ISSUE TITLE
elasticsearch caveats some updating

### ISSUE THREAD

**Issue by meeas** [Homebrew/homebrew#38141]
The first part of the caveats says:

To have launchd start elasticsearch at login:
    mkdir -p ~/Library/LaunchAgents
    ln -sfv /usr/local/opt/elasticsearch/*.plist ~/Library/LaunchAgents

To make it start at boot (instead of login) that should probably be updated to:

To have launchd start elasticsearch at login:
    mkdir -p ~/Library/LaunchAgents
    chown root /usr/local/opt/elasticsearch/*.plist
    ln -sfv /usr/local/opt/elasticsearch/*.plist /Library/LaunchAgents

Basically adding the chown command and symlinking to the main /Library tree instead of the user's ~/Library tree.  Alternately the receipe could be updated to automatically do the chown since it will probably have to be updated to maintain that ownership between elasticsearch upgrades.

**Comment by mikemcquaid** [comment]
This is intentional, we have separate instructions for things that need to be started at boot and we don't want to do that with `elasticsearch`. Thanks!
